pkg/virt-controller/watch/vmi_test.go

```diff
@@ -1605,6 +1607,59 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 
 			controller.Execute()
 		})
+		Context("And the virt-launcher pod has Multus annotations and SRIOV interface", func() {
+			DescribeTable("should copy Multus annotations to VirtualMachineInstance and change vmi status to Scheduled", func(oldStatusAnnotation string) {
+				By("creating a vmi in Scheduling phase and a SRIOV interface")
+				vmi := NewPendingVirtualMachine("testvmi")
+				setReadyCondition(vmi, k8sv1.ConditionFalse, virtv1.PodConditionMissingReason)
+				vmi.Status.Phase = virtv1.Scheduling
+				vmi.Spec.Domain.Devices.Interfaces = []virtv1.Interface{libvmi.InterfaceDeviceWithSRIOVBinding("sriov-interface")}
+				if oldStatusAnnotation != "" {
+					vmi.Annotations[networkv1.NetworkStatusAnnot] = oldStatusAnnotation
+				}
+
+				By("attaching a virt-launcher pod with Multus annotations to the vmi")
+				networkStatusAnnotation := `new-network-status`
+				pod := NewPodForVirtualMachine(vmi, k8sv1.PodRunning)
+				pod.Annotations[networkv1.NetworkStatusAnnot] = networkStatusAnnotation
+
+				addVirtualMachine(vmi)
+				podFeeder.Add(pod)
+				addActivePods(vmi, pod.UID, "")
+
+				vmiInterface.EXPECT().Update(gomock.Any()).Do(func(arg interface{}) {
+					Expect(arg.(*virtv1.VirtualMachineInstance).Annotations).To(HaveKeyWithValue(networkv1.NetworkStatusAnnot, networkStatusAnnotation))
+					Expect(arg.(*virtv1.VirtualMachineInstance).Status.Phase).To(Equal(virtv1.Scheduled))
+				})
+
+				controller.Execute()
+			},
+				Entry("and vmi has no former network-status annotation", ""),
+				Entry("and vmi has a former network-status annotation", "old-network-status-from-migrated-pod"),
+			)
+		})
+		Context("And the virt-launcher pod does not have all the Multus annotations and SRIOV interface", func() {
```

Comment:
```suggestion
		Context("VMI does not have an SR-IOV interface and the virt-launcher pod does not have the network-status annotation ", func() {
```

Reply:
but it does have an SRIOv interface

Reply:
changed to `And the VMI has an SR-IOV interface and the virt-launcher pod does not have the network-status annotation`

Reply:
My bad, sorry for the confusion.